### ISSUE TITLE
Title bar not showing the correct color

### ISSUE THREAD

**Issue by NotchApple1703** [rdbende/Sun-Valley-ttk-theme#84]
![image](https://user-images.githubusercontent.com/96767159/236256588-4b9327f2-a8ea-477b-8b53-bf0f9ccc095e.png)
When I'm using dark theme, the title bar is still white

**Comment by HuyHung1408** [comment]
@NotchApple1703 There's a working method, using [ntkutils](https://github.com/not-nef/ntkutils).
<img width="665" alt="image" src="https://user-images.githubusercontent.com/86362423/236384548-3af720b8-0097-4b7a-8292-4912929fb912.png">

**Comment by rdbende** [comment]
Yeah. Personally I wouldn't recommend using ntkutils for this, but here's the source they've taken the code from: https://gist.github.com/Olikonsti/879edbf69b801d8519bf25e804cec0aa
